modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -153,7 +160,13 @@ const ColorPicker = ({
   };
 
   return (
-    <Popup width={250} padding={Popup.Padding.s} data-testid="canvas-color-picker" {...elemProps}>
+    <Popup
+      width={250}
+      padding={Popup.Padding.s}
+      transformOrigin={transformOrigin}
+      data-testid="canvas-color-picker"
```

Comment:
@jtschult Heads up in case your team uses data-testid to query the `ColorPicker` popup in tests, we generally don't ship data attributes in Canvas Kit so not sure how this slipped through the first time. I'll open a separate PR to get rid of this.